library/profile/gmon.c

```diff
@@ -155,7 +155,8 @@ write_bb_counts(int fd) {
 
 void monstartup(uint32 low_pc, uint32 high_pc) {
     uint8 *cp;
-    uint32 lowpc, highpc, text_start;
+    uint32 lowpc, highpc;
+    uint32 text_start = 0;
```

Comment:
The `text_start` was not initialised and when `if (low_pc == 0 && high_pc == 0) {` was false this had no initial value at `p->text_start = text_start;` on line 175